simulation/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Install the simulator
 RUN wget coppeliarobotics.com/files/CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz && \
     tar -kxvf CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz && \
-    rm CoppeliaSim_Edu_V4_0_0_Ubuntu16_04.tar.xz
-COPY libsimExtROSInterface.so CoppeliaSim_Edu_V4_0_0_Ubuntu16_04/libsimExtROSInterface.so
```

Comment:
are libsimExtROSInterface.so and servertest1.ttt no longer required?

Reply:
libsimExtROSInterface.so is no longer required to be copied because it comes with the wget tarball. Servertest1 (renamed to comm_server_scene.ttt at David's request) is still required, but it needs to be able to be updated as more information needs to be communicated (e.g. controls suddenly decides it wants to use global velocity), so we're keeping it in the main repo and requiring that the files be mounted for simulation to happen. RunSim.sh finds and copies comm_server_test into the CoppeliaSim folder.